h/static/scripts/tests/controllers/search-bar-controller-test.js

```diff
@@ -51,125 +66,135 @@ describe('SearchBarController', function () {
   });
 
   afterEach(function () {
-    ctrl = null;
+    document.body.removeChild(testEl);
```

Comment:
In PhantomJS 2.x the easiest way to remove an element from the DOM is with [element.remove()](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove)